utils.js

```diff
@@ -42,5 +42,7 @@ function messageResponse (name, message) {
 
 module.exports = {
   parseMessage,
-  messageResponse
+  messageResponse,
+  printGeneralHelp,
```

Comment:
I would say making these functions public wouldn't be a right choice because that would loosen the abstraction. They are more like the private methods of Utils module.